dotCMS/src/integration-test/java/com/dotmarketing/quartz/job/IntegrityDataGenerationJobTest.java

```diff
@@ -0,0 +1,169 @@
+package com.dotmarketing.quartz.job;
+
+import com.dotcms.IntegrationTestBase;
+import com.dotcms.integritycheckers.IntegrityUtil;
+import com.dotcms.publisher.endpoint.bean.PublishingEndPoint;
+import com.dotcms.publisher.endpoint.bean.impl.PushPublishingEndPoint;
+import com.dotcms.rest.IntegrityResource;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import com.rainerhahnekamp.sneakythrow.Sneaky;
+import com.tngtech.java.junit.dataprovider.DataProviderRunner;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.quartz.JobDataMap;
+import org.quartz.JobDetail;
+import org.quartz.JobExecutionContext;
+import org.quartz.SchedulerException;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+import java.util.UUID;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @author victor
+ */
+@RunWith(DataProviderRunner.class)
+public class IntegrityDataGenerationJobTest extends IntegrationTestBase {
+
+    private IntegrityDataGenerationJob integrityDataGenerationJob;
+    private PublishingEndPoint endpoint;
+    private String requestId;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    @Before
+    public void before() throws DotDataException {
+        integrityDataGenerationJob = new IntegrityDataGenerationJob();
+        endpoint = new PushPublishingEndPoint();
+        endpoint.setId(UUID.randomUUID().toString());
+        requestId = UUID.randomUUID().toString();
+
+        DotConnect dotConnect = new DotConnect();
+        dotConnect
+                .setSQL("delete from QRTZ_EXCL_SIMPLE_TRIGGERS where TRIGGER_NAME = ? and TRIGGER_GROUP = ?")
+                .addParam(IntegrityDataGenerationJob.TRIGGER_NAME)
+                .addParam(IntegrityDataGenerationJob.TRIGGER_GROUP)
+                .loadObjectResults();
+        dotConnect
+                .setSQL("delete from QRTZ_EXCL_TRIGGERS where TRIGGER_NAME = ? and TRIGGER_GROUP = ?")
+                .addParam(IntegrityDataGenerationJob.TRIGGER_NAME)
+                .addParam(IntegrityDataGenerationJob.TRIGGER_GROUP)
+                .loadObjectResults();
+        dotConnect
+                .setSQL("delete from QRTZ_EXCL_JOB_DETAILS where JOB_NAME = ? and JOB_GROUP = ?")
+                .addParam(IntegrityDataGenerationJob.JOB_NAME)
+                .addParam(IntegrityDataGenerationJob.JOB_GROUP)
+                .loadObjectResults();
+    }
+
+    @Test
+    public void testGenerateIntegrationData() throws Exception {
```

Comment:
@victoralfaro-dotcms here we use the following convention for the test javadoc:
/**
* Method to test: include this in case the method to test does not belong to the class this test-class is testing 
* Given Scenario: Scenario under test
* ExpectedResult: Expected result that the assertion is checking against
*
*/

Reply:
Alright.